bundles/core/org.eclipse.smarthome.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/XmlThingTypeProvider.java

```diff
@@ -299,13 +101,41 @@ public void unsetConfigDescriptionProvider(ConfigDescriptionProvider configDescr
         this.configDescriptionProvider = null;
     }
 
-    @Reference(target = "(esh.scope=core.xml)")
+    @Reference(target = "(esh.scope=core.xml.channels)")
```

Comment:
I assume annotated fields does not work with the old DS we need to support. Correct?